samples/boards/nrf52/mesh/onoff_level_lighting_vnd_app/src/mesh/device_composition.c

```diff
@@ -554,18 +637,18 @@ static void gen_onpowerup_set_unack(struct bt_mesh_model *model,
 				    struct bt_mesh_msg_ctx *ctx,
 				    struct net_buf_simple *buf)
 {
```

Comment:
Wouldn't it make sense to have this function call your new `onpowerup_set()` function, so you don't need to duplicate the code? That was the refactoring that I was suggesting earlier.

Reply:
Done !!